rust/worker/src/log/log.rs

```diff
@@ -80,6 +90,36 @@ pub(crate) struct GrpcLog {
 }
 
 impl GrpcLog {
+    pub(crate) fn try_wrap_request_with_trace_context<T>(
+        &self,
+        request: &mut Request<T>,
+        spn: &tracing::Span,
+    ) {
+        // If span is disabled then nothing to append in the header.
+        if spn.is_disabled() {
+            return;
+        }
+        let metadata: &mut MetadataMap = request.metadata_mut();
+        let span_id = spn.context().span().span_context().span_id().to_string();
+        let trace_id = spn.context().span().span_context().trace_id().to_string();
+        println!("Span id {}, trace id {}", span_id, trace_id);
+        let ascii_traceid = MetadataValue::from_str(trace_id.as_str());
+        let ascii_spanid = MetadataValue::from_str(span_id.as_str());
+        // Errors are not fatal.
+        match ascii_traceid {
+            Ok(id) => {
+                metadata.append("chroma-traceid", id);
```

Comment:
is this constant across services - should we move it out into a const?

Reply:
yep, don't want to hold this PR for that?